config/initializers/devise.rb

```
@@ -0,0 +1,313 @@
+# breaking changes in upgrades (i.e., in the event that future versions of
+# Devise change the default values for those options).
+#
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
+  # The secret key used by Devise. Devise uses this key to generate
+  # random tokens. Changing this key will render invalid all existing
+  # confirmation, reset password and unlock tokens in the database.
+  # Devise will use the `secret_key_base` as its `secret_key`
+  # by default. You can change it below and use your own secret key.
+  # config.secret_key = 'f5c3627ab84eb5a12eab9feae096d10ac001e2a3980e87567922107429a516f3939c17512beb6c2b57330814611d42511d91d603a058391729519332b0a3e826'
```

rubocop (1.69.1)

[convention] Line is too long. [154/120]

(Layout/LineLength)